lib/runtime/ppx_mysql_runtime.ml

```diff
@@ -1,3 +1,57 @@
+type column_error =
+  [ `Expected_non_null_column of int * string
+  | `Deserialization_error of int * string * string * string * string ]
+
+type 'a deserializer = string -> ('a, [`Deserialization_error of string]) result
+
+let wrap_failure of_string s =
```

Comment:
Wouldn't `f` be more appropriate here? Or maybe a type signature restricting `of_string` to convert?

Reply:
I've restricted the signature.